R/check_ss_ssdtc_dead_ds.R

```diff
@@ -84,9 +90,14 @@ check_ss_ssdtc_dead_ds <- function(SS, DS) {
         fail(lacks_msg(DS, c("USUBJID", "DSSTDTC", "DSDECOD", "DSCAT")))
 
     }else{
+        
+        #Apply company specific preprocessing function
+        DS = preproc(DS,...)
 
         myss <- subset(SS, !is_sas_na(SS$SSDTC) & toupper(SS$SSSTRESC) == 'DEAD', c("USUBJID","SSDTC","SSSTRESC", "VISIT"))
-        myds <- subset(DS, !is_sas_na(DS$DSSTDTC) & toupper(DS$DSDECOD) == 'DEATH' & toupper(DS$DSCAT) == "DISPOSITION EVENT", c("USUBJID", "DSSTDTC", "DSDECOD", "DSCAT"))
+        myds <- subset(DS, !is_sas_na(DS$DSSTDTC) & toupper(DS$DSDECOD) == 'DEATH' & toupper(DS$DSCAT) == "DISPOSITION EVENT") %>% 
```

Comment:
@sarabodach what do you think of removing the `DSCAT=DISPOSITION EVENT` filtering here?  In one study I see patients who discontinued study due to non-death reason and their post study reporting of death is in DS with DS.DSCAT=OTHER.  This check flags a false positive on these patients

Reply:
lets keep it as is